### ISSUE TITLE
Task: Get user coordinates

### ISSUE THREAD

**Issue by MoribundMedium** [nashvillefcc/local-weather-app#28]
# Parent Feature
#3 Display Local Weather

## Blockers
N/A

## Acceptance Criteria
Request access to the user's location coordinates and store it in a variable for use in other tickets. Possible implementation with [the Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API).

If the user declines to share location, the UI should display a message saying something to the effect of "Unable to display weather without location data. Please refresh to allow location services on your browser [or input a location here (depending on feature #34)]."

**Comment by jhadams82** [comment]
I couldn't find a branch specifically for this task. Was it already merged into "develop" and the branch deleted? If so, it would seem we didn't meet the acceptance criteria. We logged the user's coords to the console, but didn't store them in a variable for later use. branch 27 seems to have this implemented well. maybe we can use some of the code there?

**Comment by jhadams82** [comment]
yeah, the current version of FEATURE/display-local-weather has this functionality so maybe this task should be moved from "pull request/ready for review" to "done"
